easybuild/easyconfigs/n/Nextflow/Nextflow-19.12.0.eb

```diff
@@ -0,0 +1,47 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	Apache v2.0
+#
+# Notes::	Adapted from Author: @J. Sassmannshausen (Crick HPC team)
+##
+
+easyblock = 'CmdCp'
+
+name = 'Nextflow'
+version = '19.12.0'
+
+homepage = 'https://www.%(namelower)s.io/'
+description = """Nextflow is a reactive workflow framework and a programming DSL 
+ that eases writing computational pipelines with complex data"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/%(namelower)s-io/%(namelower)s/archive']
+sources = ['v%(version)s-edge.tar.gz']
+checksums = ['16d38a14aeab636531da6d5d1894898d13a359c5f699eefcfb433fc3340e1b8a']
+
+# specify dependency on Java/11 "wrapper", rather than a specific Java version
+dependencies = [('Java', '11', '', True)]
+
+skipsteps = ['configure', 'build']
```

Comment:
@robqiao Same here, better use the `Binary` easyblock and drop this line?

Reply:
Agreed and fixed 👍  issue #9730 seems a duplicate on this

Reply:
In that case `CmdCp` makes sense, thanks for clarifying @robqiao!